src/zig_clang.cpp

```diff
@@ -3245,6 +3245,17 @@ double ZigClangFloatingLiteral_getValueAsApproximateDouble(const ZigClangFloatin
     return casted->getValueAsApproximateDouble();
 }
 
+void ZigClangFloatingLiteral_getValueAsApproximateQuadBits(const ZigClangFloatingLiteral *self, uint64_t *low, uint64_t *high) {
+    auto casted = reinterpret_cast<const clang::FloatingLiteral *>(self);
+    llvm::APFloat apf = casted->getValue();
+    bool ignored;
+    apf.convert(llvm::APFloat::IEEEquad(), llvm::APFloat::rmNearestTiesToEven, &ignored);
+    const llvm::APInt api = apf.bitcastToAPInt();
+    const uint64_t *api_data = api.getRawData();
+    *low = api_data[0];
+    *high = api_data[1];
+}
```

Comment:
This is similar to how Clang/LLVM implements `getValueAsApproximateDouble()` (except here the bitcasting is handled in Zig and not C++).
- https://github.com/llvm/llvm-project/blob/1c6c01fbde87171457a5bce7b147fa5bcbaddae7/clang/lib/AST/Expr.cpp#L1050-L1059
- https://github.com/llvm/llvm-project/blob/1c6c01fbde87171457a5bce7b147fa5bcbaddae7/llvm/lib/Support/APFloat.cpp#L3666-L3671
- https://github.com/llvm/llvm-project/blob/0e06ddf0f6896cfd817a1b97a43b78331e0b1d66/llvm/include/llvm/ADT/APInt.h#L1659-L1664